media-classification/src/main/java/nl/vpro/domain/classification/LocalizedString.java

```diff
@@ -2,10 +2,14 @@
 
 import java.util.Locale;
 
-import static nl.vpro.i18n.Locales.score;
+/**
+ * Extension of {@link nl.vpro.i18n.LocalizedString} in the correct XML namespace
+ */
 
 public class LocalizedString extends nl.vpro.i18n.LocalizedString {
-  public static LocalizedString copy(nl.vpro.i18n.LocalizedString copy) {
+    private static final long serialVersionUID = -6545505135867762847L;
```

Comment:
Interesting pick. Waar komt dit getal vandaan?

Reply:
Heeft intellij voor me bedacht. Ik neem aan dat het de huidige impliciete waarde is.